packages/shared/src/components/CombatReport/CombatReplay/ReplayEvents/events/ReplayHpUpdateEvent.tsx

```diff
@@ -9,18 +9,17 @@ interface IProps {
 }
 
 export function ReplayHpUpdateEvent(props: IProps) {
+  const isCrit = props.event.isCritical;
   return (
     <ReplayEventContainer {...props}>
       <ReplayEventSpellInfo event={props.event} direction="right" expanded={props.expanded} />
       {props.expanded && <div className="ml-0.5">and dealt</div>}
       <div className="ml-0.5">
         <div className={`${props.event.amount >= 0 ? 'text-success' : 'text-error'} font-bold`}>
+          {isCrit ? '*' : ''}
           {props.event.amount > 0 ? '+' : ''}
           {props.event.effectiveAmount}
-          {
-            // TODO: implement isCrit in parser
-            // {isCrit(props.event, context.combat.wowVersion) ? '*' : ''}
-          }
+          {isCrit ? '*' : ''}
```

Comment:
I think having * at the end should be sufficient for differentiation. We'll probably want to have a consistent indicator in the death view as well, and the space is super limited there.